news/2022-02-23-new-featured-artist-rabbit-house.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Rabbit House"
+date: 2022-02-23 09:00:00 +0000
+---
+
+You might not recognize our latest Featured Artist by their name, but you *definitely* recognize **Rabbit House**'s beatmaps.
+
+![](https://assets.ppy.sh/artists/242/header.jpg)
+
+**Rabbit House**, also known as [Nakagawa-Kanon](https://osu.ppy.sh/users/87065) or *dksslqj* on osu!, is best known among circle clickers for creating one of the game's most renowned beatmaps of all time: [xi - FREEDOM DiVE](https://osu.ppy.sh/beatmapsets/39804).
+
+Since that legendary beatmap was ranked a decade ago, **Rabbit House** has developed a musical skillset to match their rhythm game expertise. **Rabbit House**'s blood-pumping music recently made a splash in games like *Lanota* and *Arcaea*, and has finally come full-circle by returning officially to osu! as of today.
+
+We have **23** tracks, all available on [**Rabbit House**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/242). And if that weren't enough, one of them is being released exclusively on osu! too. Check them all out if you want to be a part of this beatmapper-turned-musician's story!
+
+Haven't heard **Rabbit House**'s tunes before? The video below showcases an intersection between worlds: **Rabbit House**'s beatmap of their own song. Check it out along with the rest of the previews below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/242/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Have A Party Time!
+
+Try playing [this osu! map](https://osu.ppy.sh/beatmapsets/652751) created by [Nakagawa-Kanon](https://osu.ppy.sh/users/87065) (a.k.a. **Rabbit House**!)
```

Comment:
```suggestion
Try playing [this osu! map](https://osu.ppy.sh/beatmapsets/652751) created by [Nakagawa-Kanon](https://osu.ppy.sh/users/87065) (a.k.a. **Rabbit House**) himself!
```

Reply:
post doesn't mention gender, may need to use "themselves" if it should be unspecified

Reply:
fine as it is honestly